### ISSUE TITLE
Inconsistent performance w/ `fasttext` itself

### ISSUE THREAD

**Issue by bkj** [salestock/fastText.py#108]
`fasttext` treats the EOS character `</s>` as a token. `fastText.py` does not, so the results are inconsistent.

```
# data.txt
__label__pos a a a
__label__neg b b b
__label__pos a a a
__label__neg b b b
__label__pos a a a
__label__neg b b b
__label__pos a a a
__label__neg b b b

# train.py
import fasttext as ft
classifier = ft.supervised('data.txt', 'model', epoch=100)

# test.py
import fasttext as ft
classifier = ft.load_model('./model.bin')
classifier.predict_proba(open('data.txt').read().splitlines(), 2)

[[(u'pos', 0.998047), (u'neg', 1.95313e-08)],
 [(u'neg', 0.998047), (u'pos', 1.95313e-08)],
 [(u'pos', 0.998047), (u'neg', 1.95313e-08)],
 [(u'neg', 0.998047), (u'pos', 1.95313e-08)],
 [(u'pos', 0.998047), (u'neg', 1.95313e-08)],
 [(u'neg', 0.998047), (u'pos', 1.95313e-08)],
 [(u'pos', 0.998047), (u'neg', 1.95313e-08)],
 [(u'neg', 0.998047), (u'pos', 1.95313e-08)]]

# test.sh
fasttext predict-prob ./model.bin data.txt 2

__label__pos 0.990234 __label__neg 0.00781252
__label__neg 0.990234 __label__pos 0.00781252
__label__pos 0.990234 __label__neg 0.00781252
__label__neg 0.990234 __label__pos 0.00781252
__label__pos 0.990234 __label__neg 0.00781252
__label__neg 0.990234 __label__pos 0.00781252
__label__pos 0.990234 __label__neg 0.00781252
__label__neg 0.990234 __label__pos 0.00781252
```

Note that if the last character in `data.txt` is _not_ `\n`, then the prediction for the last line will be
```
__label__neg 0.998047 __label__pos 1.95313e-08
```
which is the prediction if we drop the EOS character.  This is obviously a bug in `fasttext` itself, but is helps illustrate the issue.  

A (sortof dirty) fix would be to edit these lines in `model.py`:
```
    def predict(self, texts, k=1):
        return [self._model.classifier_predict(text + ' </s>', k,
                    self.label_prefix, self.encoding) for text in texts]

    def predict_proba(self, texts, k=1):
        return [self._model.classifier_predict_prob(text + ' </s>', k,
                    self.label_prefix, self.encoding) for text in texts]
```

**Comment by mktal** [comment]
I had the same issue and created a pull request #107

**Comment by pyk** [comment]
This problem should be fixed on [0.8.3](https://pypi.python.org/pypi/fasttext/0.8.3)
